tests/test_query.py

```diff
@@ -782,7 +779,7 @@ async def test_describe_select(
             ],
         ),
         (
-            "select database(), schema(), left(user(),instr(concat(user(),'@'),'@')-1)",
+            "select database(), schema(), left(user(),instr('@', concat(user(),'@'))-1)",
```

Comment:
I think this isn't right for MySQL: https://dev.mysql.com/doc/refman/8.4/en/string-functions.html#function_instr

A lot of these fixtures come from standard mysql clients, so we have to conform to them.

So instead of changing the test, I think we need to fix the sqlglot parser